tools/lang-handler-aliases.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+/**
+ * For each lang-*.js file, runs the language handler in a fake context where
+ * PR.registerLangHandler collects handler names without doing anything else,
+ * and then makes copies of the JS extension under all registered language
+ * names lang-<EXT>.js
+ *
+ * As verbose output, it also prints lines of the form:
+ *    lang-foo.js => foo,bar,baz
+ */
+
+var fs = require('fs');
+var path = require('path');
+var vm = require('vm');
+
+function fileExists(filePath) {
+    try {
+        return fs.statSync(filePath).isFile();
```

Comment:
creating symlinks for the aliases would be better than making copies, but I'm not sure if they are supported on all Windows versions..